app/tests/spec/lib/cropper.js

```diff
@@ -77,8 +80,14 @@ function (chai, RouterMock, CanvasMock, p, Cropper, EphemeralMessages, View,
 
     describe('with a cropper', function () {
       var cropper;
+      var mockMetrics;
 
       beforeEach(function () {
+        mockMetrics = {
+          logEvent: function (event) {
```

Comment:
This unused `event` here was causing the JSHint error/warning that broke the Travis build. Since it isn't doing anything here, we can probably just drop the `event` parameter and the `logEvent()` is just a noop.